### PR TITLE
Preparing release 2.2.6

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: bigquery/CHANGES.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### 1.2.1 - 2021-XX-XX
+### 1.2.1 - 2021-04-14
```
Comment (Deependra-Patel) [inline bigquery/CHANGES.md:1]: You mean 2022?

Comment (Deependra-Patel) [reply bigquery/CHANGES.md:1]: Not sure about previous dates as well.

File: gcs/CHANGES.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-### 2.2.6 - 2022-XX-XX
+### 2.2.6 - 2022-04-14
+
+1. Improve trace logging for gRPC channels.
+
+2. Bug fixes.
```
Comment (Deependra-Patel) [inline gcs/CHANGES.md:5]: We should mention bug fixes in detail as the write one was important.

@mprashanthsagar should be able to give summary for here.

File: bigquery/CHANGES.md (hunk 1 above)
Comment (mprashanthsagar) [reply bigquery/CHANGES.md:1]: Do we have changes to bigquery connector ? We can skip releasing bigquery hadoop connector

**Review by mprashanthsagar — CHANGES_REQUESTED** [review]

**Review by mprashanthsagar — APPROVED** [review]

**Pull request merged**